src/main/java/be/seeseemelk/mockbukkit/inventory/meta/ItemMetaMock.java

```diff
@@ -252,6 +259,7 @@ public void assertHasNoLore() throws AssertionError {
 	    map.put("customModelData", this.customModelData);
 	    map.put("persistentDataContainer", this.persistentDataContainer);
 	    map.put("damage", this.damage);
+        map.put("repairCost", this.repairCost);
```

Comment:
indentation

Reply:
Actually, you'll find that the indentation there is a mix of spaces and tabs. If you read the bottom of the PR I mention exactly why this is how it is.

Reply:
Normally 4-spaces should be the standard for this project, if you find any inconsistencies in the file, feel free to correct it.
We really need a CONTRIBUTING.md, checksytel or .editorconfig file for these style guidelines at some point.